x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -1,5 +1,29 @@
 {
   "properties": {
+    "actions": {
+      "properties": {
+        "count_total": {
+          "type": "long"
+        },
+        "count_active_total": {
+          "type": "long"
+        },
+        "count_by_type": {
+          "properties": {
+            "DYNAMIC_KEY": {
+              "type": "long"
+            }
+          }
+        },
+        "count_active_by_type": {
+          "properties": {
+            "DYNAMIC_KEY": {
+              "type": "long"
+            }
```

Comment:
@mindbat for now, when there is a dynamic key with unknown values, `schema` will be reported with `DYNAMIC_KEY`. Would that work for you?

Reply:
I'm afraid I don't understand the question. This is the schema internal to Kibana? So there's no direct correspondence to what gets sent in the object to be indexed?

Reply:
@mindbat that's not quite true. The schema is a way of indicating what is sent in the object and the field type we can expect to map that as (long vs float, date, keyword etc). The schema won't necessarily correspond directly to the index mapping but it will indicate the value type for that field.

That being said, the point of introducing schemas was to provide _known_ fields and their expected types. We (from the Kibana side) shouldn't provide anything that we, um, don't know!

@afharo we'll still need to update mappings for fields we _do_ know and are certain about and I'm  we're highly unlikely to be maintaining mappings for dynamic fields now anyway (how could we if we don't know what they are 🤷‍♀️ ). 

@mindbat Would you be ok with perhaps ignoring the mappings for these types of "DYNAMIC_KEY" fields? i.e. we maintain and update diffs in the fields we know but don't concern ourselves (i.e. don't map them at all) with those that we don't?

Reply:
@mindbat everything reported in the `schema` is directly correlated to the data that gets shipped under `stack_stats.kibana.plugins`.

Unfortunately, we've still got a few objects whose keys are dynamic and we need the `schema` to describe them in some way. Then it's up to the Remote Telemetry Service to map them via Dynamic Mapping Templates, change their structure in the Indexer, or don't index/map them at all. Our take up until now has been: index them but don't map them (so they are in the `_source` for advanced visualisations like Vega, but can't really be searched/aggregated).

The question is: is `DYNAMIC_KEY` a valid indicator for you to identify these type of keys?

FYI: Based on the discussion in here (https://github.com/elastic/kibana/pull/78832#discussion_r497766525), we can provide some of the already-known keys. But the list is likely to grow dynamically (and we'll try to keep it up to date). Should, in those cases, still provide the known keys and the extra `DYNAMIC_KEY` for you to be aware of that case?